x/migration/module/cmd/validate_morse_accounts.go

```diff
@@ -182,33 +201,31 @@ func runValidateMorseAccounts(_ *cobra.Command, args []string) error {
 
 	// Sort the morse claimable accounts for use in binary search.
 	sortedMorseClaimableAccounts := msgImportMorseClaimableAccounts.MorseAccountState.Accounts
+	numSortedMorseClaimableAccounts := len(sortedMorseClaimableAccounts)
 	sortMorseClaimableAccounts(msgImportMorseClaimableAccounts.MorseAccountState.Accounts)
 
-	// Check that each given morse address exists in the morse account state
-	// and print each for inspection.
+	// Check that each given morse address exists in the morse account state and print each for inspection.
 	missingMorseAddresses := make([]string, 0)
 	for _, targetMorseAddress := range morseAddresses {
 		// Normalize the morse address to uppercase hex.
 		targetMorseAddress = strings.ToUpper(targetMorseAddress)
 
 		// Use binary search to find the index of the target morse address efficiently.
-		morseAccountIdx := sort.Search(len(sortedMorseClaimableAccounts), func(i int) bool {
+		morseAccountIdx := sort.Search(numSortedMorseClaimableAccounts, func(i int) bool {
 			ithMorseSrcAddress := sortedMorseClaimableAccounts[i].GetMorseSrcAddress()
 			return ithMorseSrcAddress >= targetMorseAddress
 		})
 
 		// DEV_NOTE: The index returned when the target is not found is the length of the slice.
-		if morseAccountIdx >= len(sortedMorseClaimableAccounts) {
+		if morseAccountIdx >= numSortedMorseClaimableAccounts {
 			missingMorseAddresses = append(missingMorseAddresses, targetMorseAddress)
-
-			// Morse address not found, move on to the next one.
+			logger.Logger.Warn().Msgf("Morse address %s not found in the morse account state. Moving on to the next one.", targetMorseAddress)
```

Comment:
We don't want log this in the middle of the loop. The point of collecting the imto `missingMorseAddresses` is so that they can be formatted and printed later.
```suggestion
			// Morse address not found, move on to the next one.
```